challenge.rb

```diff
@@ -45,7 +45,15 @@ def okay_to_drink? age, drinking_age
 #   "Fix the AC whenever you have the chance. It's cool."
 
 def ac_monitor current_temp, ac_working, desired_temp
-  # Replace this comment with your code.
+  if ((current_temp > desired_temp) && (ac_working == true))
+    p "Turn on the AC."
+  elsif ((current_temp <= desired_temp) && (ac_working == true))
+    p "Just right!"
+  elsif ((current_temp > desired_temp) && (ac_working != true))
+    p "Fix the AC now! It's hot!"
+  else
+    p "Fix the AC whenever you have the chance. It's cool."
+
```

Comment:
Should have been `return` rather than `p` - maybe we want to use the result of `ac_monitor` to dive another program!